assets/less/app.less

```diff
@@ -1,5 +1,5 @@
 @import 'npm://font-awesome/less/font-awesome';
-@import 'npm://highlightjs/styles/tomorrow';
+@import 'npm://highlight.js/src/styles/tomorrow';
```

Comment:
@Dignifiedquire I don't think we are keeping any license in the app.js file. is there an option for this? or should i create an issue to update all the licenses to license.